web/src/beta/features/Editor/index.tsx

```diff
@@ -35,8 +35,7 @@ const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab }) => {
               direction="vertical"
               gutter="end"
               size={metrics.propertyMenuMinWidth}
-              minSize={metrics.propertyMenuMinWidth}
-              maxSize={metrics.propertyMenuMaxWidth}>
+              minSize={metrics.propertyMenuMinWidth / 4}>
```

Comment:
I don't think we need `minSize` anymore since it doesn't really make sense. Instead, remove this and just pass the `initialSize` prop. (BTW, rename the prop initialSize please. We rename it in the Resizable func but not sure why the prop isn't just initialSize to begin with 👍🏻